app/controllers/employees_controller.rb

```diff
@@ -11,25 +12,29 @@ def show
 
   def new
     @employee = Employee.new
-    @private_number = rand(111111..999999)
-    if Employee.exists?(private_number: @private_number)
-      @private_number = rand(111111..999999)
-    end
```

Comment:
You are removing changes that you just added in a previous commit. I suggest making all the private number changes in the same commit.

Reply:
Ok